hazelcast/src/main/java/com/hazelcast/json/internal/JsonSchemaStructNode.java

```diff
@@ -89,7 +99,38 @@ public int hashCode() {
     @Override
     public String toString() {
         return "JsonSchemaStructNode{"
-                + "inners=" + inners
-                + '}';
+            + "inners=" + inners
+            + '}';
+    }
+
+    @Override
+    public void writeData(ObjectDataOutput out) throws IOException {
+        out.writeInt(inners.size());
+        for (JsonSchemaNameValue nameValue : inners) {
```

Comment:
Minor one not to create meta iterator object:
```suggestion
		for (int i = 0; i < inners.size(); i++) {
            out.writeObject(inners.get(i));
        }
```